web/client/components/manager/users/UserGroups.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+// const Message = require('../I18N/Message');
+const Select = require('react-select');
+const Message = require('../../I18N/Message');
+
+
+// const ConfirmModal = require('./modals/ConfirmModal');
+// const GroupManager = require('./GroupManager');
+
+const UserCard = React.createClass({
+    propTypes: {
+        // props
+        groups: React.PropTypes.array,
+        onUserGroupsChange: React.PropTypes.func,
+        user: React.PropTypes.object
+    },
+    getDefaultProps() {
+        return {
+            groups: [],
+            onUserGroupsChange: () => {},
+            user: {}
+        };
+    },
+
+    onChange(values) {
+        if (values === null) {
+            return;
+        }
+        this.props.onUserGroupsChange("groups", values.map((group) => {
+            let index = this.props.groups.findIndex((availableGroup)=>availableGroup.id === group.value);
+            return index >= 0 ? this.props.groups[index] : null;
+        }).filter(group => group));
+    },
+    getEveryOneGroups() {
```

Comment:
With that name it seems you can have many everyone groups? Is it true, or you always get an array of exactly one element?